.github/workflows/meeting-reminder.yaml

```diff
@@ -49,13 +49,11 @@ jobs:
           message: |-
             Liebe Freifunk-Interessent:innen,
 
-            unser nächstes Treffen findet am kommenden Montag (${{ needs.date.outputs.next_date }}) um 19:30 Uhr in den Räumen
-            des Netz39 e.V. (Leibnizstr. 32) statt.
+            unser nächstes Treffen findet am kommenden Montag (${{ needs.date.outputs.next_date }}) um 19:30 Uhr in den Räumen des Netz39 e.V. (Leibnizstr. 32) statt.
 
-            Falls ihr hybrid teilnehmen möchtet, gebt uns bitte bis 2h vorher per Mail an kontakt@md.freifunk.net
-            Bescheid, dann kümmern wir uns um die Online-Variante.
+            Falls ihr hybrid teilnehmen möchtet, gebt uns bitte bis 2h vorher per Mail an kontakt@md.freifunk.net Bescheid, dann kümmern wir uns um die Online-Variante.
 
             Viele Grüße,
             Das ffmd-Team
         env:
-          APPRISE_URL: mailtos://mail.netz39.de/?user=${{ secrets.APPRISE_MAIL_USER }}&pass=${{ secrets.APPRISE_MAIL_PASS }}&to=ffmd@netz39.de,ffmd-dev@netz39.de,announce@netz39.de
+          APPRISE_URL: mailtos://mail.netz39.de/?user=${{ secrets.APPRISE_MAIL_USER }}&pass=${{ secrets.APPRISE_MAIL_PASS }}&from=FFMD<bot@netz39.de>&to=ffmd@netz39.de,ffmd-dev@netz39.de,announce@netz39.de
```

Comment:
Mit dem from Parameter hatte ich schon rumexperimentiert. Das hat, wenn ich mich richtig erinnere, nicht funktioniert.

Reply:
Hm.... interessant. Ich hatte den von dort: https://github.com/caronc/apprise/wiki/Notify_email#parameter-breakdown

Reply:
Wollen wir es trotzdem probieren?
(Du hattest ja approved)

Reply:
Ja, weil ich dachte, du hast es getestet 😅

Reply:
Merge anyway?

Reply:
das schlimsmte awas passieren kann, ist dass der nächste Lauf keine Mails verschickt. Insofern go for it @penguineer

Reply:
Ich teste das noch kurz, gleich nach meiner anderen Deadline.